LSL.py

```diff
@@ -144,7 +157,8 @@ def __save_collected_data(path: str):
 
                 # Convert collected data to a DataFrame, format with columns above, and write to CSV
                 df = pd.DataFrame(LSL.collected_data[stream_type], columns=columns)
-                df['Timestamp'] = pd.to_datetime(df['Timestamp'], format='mixed')#'%Y-%m-%d %H:%M:%S.%f')
+                # Formatting breaks everything for some reason:
+                # df['Timestamp'] = pd.to_datetime(df['Timestamp'], format='%Y-%m-%d %H:%M:%S.%f')
```

Comment:
If this is old code it should probably be deleted. Commented out code is generally avoided.